netbox/netbox/forms.py

```diff
@@ -7,6 +7,7 @@
     ('Circuits', (
         ('provider', 'Providers'),
         ('circuit', 'Circuits'),
+        ('providernetwork', 'Provider Networks'),
```

Comment:
While this fixes the immediate issue, it does nothing to prevent future models from also being inadvertently omitted from the list. A more complete solution would be to dynamically render the dropdown list from the `SEARCH_TYPES` dictionary. Would you like to take a shot at coming up with something?

Reply:
Okay, I added some code that adds a layer of hierarchy to the `constants.py` and then builds the choices from that.